exercises/HodlCapsule.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity 0.4.24;
 
 contract Capsule {
 
-    //declare the following variables: owner_, amount_ and unlocktime_
+    //declared variables: owner_, amount_ and unlocktime_
```

Comment:
same as above if these are going to be there for them I don't think the comment is required